lib/util/common.js

```diff
@@ -41,10 +41,17 @@ function addHook (fastify) {
   }
 }
 
-function shouldRouteHide (schema, hiddenTag) {
+function shouldRouteHide (schema, hiddenTag, hideUntagged) {
```

Comment:
I'd rather pass the option object rather than adding another parameter. 3 parameters is still acceptable in conventions but as those 2 last parameters belongs from the same object; to me it would be better to factorize

Reply:
I got your point. No problem to factorize.

Do you prefer to pass the entire opts object or create a new one that includes just the shouldRouteHide options?

In the feature if the shouldRouteHide need to accept more parameters we can use that options object instead modify the plugin one.

Reply:
@zekth I committed with my preferred solution. I used a new option object to avoid passing the main plugin option.

If it doesn't match your requirements, I'll update.